core/pkg/eval/string_comparison_evaluation.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/open-feature/flagd/core/pkg/logger"
 )
 
+const (
+	StartsWithEvaluationName = "starts_with"
```

Comment:
Not really related to this PR but elsewhere camelCase is used for JSON keys, e.g. the now deprecated `fractionalEvaluation`. It would be nice to standardize I think.

Reply:
That's true but oddly enough, json logic uses snake_case for their operators: https://jsonlogic.com/operations.html#missing_some

Go figure.